walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -37,31 +37,76 @@
          return eventsAsTimeRange;
       }
     }
+    // convert events to list representation
+    List<Event> eventsAsList = new ArrayList<Event>();
+    for (Event event: events) {
+      eventsAsList.add(event);
+    }
+    
```

Comment:
i believe you can do `List<Event> listOfEvents = new ArrayList<Evenet>(events);`

Reply:
oh nice!!